### DISCUSSION TITLE
Lista de asistencia programacion 1 segunda matricula

### DISCUSSION THREAD

**Discussion by franvenero** [mmasias/24-25-PRG1#129] [General]
Buenas tardes Manuel, te dejo aquí la lista de gente de segunda matricula que quieren o no asistir a las clases de programación 1 como ya habíamos hablado. Estas son las personas que he conseguido localizar que cursan otra vez la asignatura, hay más pero no sé quienes son ni he podido dar con ellos.

La gente que **no** quiere asistir a las clases de la asignatura son: 

- Francisco Venero @franvenero
- Miguel Andecochea @Andecochea
- Hector Ares @hectorares3
- Mario Viadero @maariiooo
- Javier Rodriguez @jerdier
- Oscar Pineda @OscarPineda9
- Raul Tascon @Raultascon
- Jose Alejandro Cabrera 

La gente que **si** quiere asistir a las clases de la asignatura son: 

- Kelvia Neves @KelviaBarros05

**Comment by LavinAguado** [comment]
Buenas tardes Manuel,
Siguiendo el hilo de la conversación de mi compañero, que ha podido ponerse en contacto conmigo, digo por este medio que no asistiré a clase. 
Atentamente Daniel Lavín, alumno tercera matrícula.

**Comment by mmasias** [comment]
De acuerdo. En virtud de lo comentado, estar al pendiente del avance de la asignatura para presentar adecuadamente los retos que se vayan planteando.

**Comment by mmasias** [comment]
Para empezar, salvo @LavinAguado, ninguno de los indicados en el mensaje de @franvenero ha entregado en condiciones el reto 001, con lo que tienen un cero en dicho reto. Les animo a estar al pendiente y enviar bien los retos posteriores.

**Reply by franvenero** [reply]
Buenas Masías, el otro día en clase te comentamos @andecochea y yo que habíamos subido el reto-001 en el tiempo especificado, y posteriormente lo subimos corregido debido a que nos dijiste que lo corrigiésemos  ya que habíamos creado una carpeta que no tenía que estar. Sin embargo todavía no nos has cerrado el pull request y por eso el reto no está bien completado

**Reply by mmasias** [reply]
Entonces no estaba bien @franvenero. Las entregas son buenas por dos factores: tiempo y forma. Y más todavía para un reto tan básico. Lo dicho: les animo a estar al pendiente y enviar **bien** los retos posteriores.
